packages/flutter/lib/src/widgets/image.dart

```diff
@@ -399,6 +399,32 @@ class Image extends StatefulWidget {
   /// In the case where the network image is on the Web platform, the [cacheWidth]
   /// and [cacheHeight] parameters are ignored as the web engine delegates
   /// image decoding to the web which does not support custom decode sizes.
+  ///
+  /// ### Cross-origin images on Web
+  ///
+  /// Due to browser restriction on Cross-Origin Resource Sharing (CORS),
+  /// Flutter on the Web platform can not fetch images from other origins
+  /// (domain, scheme, or port) than the origin that hosts the app, unless the
+  /// image hosting origin explicitly allows so. CORS errors can be resolved
+  /// by configuring the image hosting server. More information on CORS can be
+  /// found at
+  /// [CORS errors](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors).
+  ///
+  /// In case that configuring the host is not available, such as when images
```

Comment:
nit: s/In case that configuring the host is not available/If it's not possible to configure the host/